gammapy/irf/edisp/core.py

```diff
@@ -293,6 +293,12 @@ def plot_bias(
     def peek(self, figsize=(15, 5)):
         """Quick-look summary plots.
 
+        This method creates a figure with three subplots:
+
```

Comment:
migration is not in the glossary, this is a jargon. And we are using either migration or energy dispersion, which might be confusing.